src/server/handlers/universal.js

```diff
@@ -52,6 +52,12 @@ export const handleMatch = (req, res, error, redirectLocation, renderProps) => {
       return;
     }
 
+    if (req.session.ssoDischarge) {
+      // Tell the client that it can pick up a discharge macaroon from the
+      // session and move it to local storage.
+      initialState.authStore = { hasDischarge: true };
```

Comment:
I thought about this, but it doesn't quite work because we need to prove that the client has had a chance to save it to local storage before deleting it.

We could perhaps push it into the initial state, it's true, but I have a slight preference for secrets not showing up in view-source, which is perhaps irrational in this case ...